containers/datalab/Dockerfile.in

```diff
@@ -115,6 +118,16 @@ RUN cd /datalab/lib/GCPData-0.1.0 && python setup.py install && \
     cd /datalab/web && /tools/node/bin/npm install && \
     cd /
 
+# Link stuff needed for nbconvert to a location where Jinja will find it.
```

Comment:
Sure, that works as well for now ... I feel its a bit short term (eg. if we add a file, we have to change it here), but we can defer it. I'd hope a script can do something like foreach file in dir1, create a link.